Gopkg.lock

```diff
@@ -3,7 +3,7 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/ContextLogic/goServiceLookup"
```

Comment:
This package does not exists :( in both orgs

Reply:
I just search and replaced. Is `github.com/ContextLogic/goServiceLookup` an internal project?

Reply:
yep :(

Reply:
I'll leave that up to you guys to fix then. ;)